conbench/tests/app/test_compare.py

```diff
@@ -39,6 +39,41 @@ def test_flash_messages(self, client):
         assert "cannot perform comparison:" in response.text
 
 
+class TestCompareBenchmarks(_asserts.AppEndpointTest):
```

Comment:
Is this the first set of tests exercising the UI's compare view?

Can we rename this to TestCompareBenchmarkResults?

Reply:
The class testing the view was already renamed `TestCompareBenchmarkResults` at some point; see L18-19.

This is a lighter-weight copy of that test that mostly just tests the redirect.